### ISSUE TITLE
An error occurs when use the onnx model to inference: axis > = 0 & axis < nbdims, because pytorch is not compatible with tensorRT7.

### ISSUE THREAD

**Issue by ssmallDream** [dusty-nv/jetson-inference#708]
Hello,
When I use the onnx model to inference on Jetson AgX, an error occurs: 
```
ERROR: onnx2trt_utils.hpp:347 In function convert_axis:
[8] Assertion failed: axis > = 0 & axis < nbdims.
[TRT] failded to parse ONNX model '/home/admin/jetson-inference/python/training/detection/pytorch-ssd/models/mydatasets/ssd-mobilenet.onnx'
```

The problem is the train code by pytorch is not compatible with the inference code of tensorRT.

When I use resnet18 for classification, I need to change "reshape()" to "flatten()" of network forward in torchvision lib, and the problem is solved: 
```
# x=x.reshape(1,-1)
x=x.flatten(1)
```

In fact, the torch vision problem has been solved here #370 , just recompile the torchvision and then retrain the model.

But when I use mobile-SSD, the same problems occur. How do I need to solve them?
https://github.com/dusty-nv/pytorch-ssd/blob/01e0146eb835860cac71763727530d463af67894/vision/ssd/ssd.py#L103
Here, we need to transform the shape of the tensor: torch.Size ([4, 19, 19, 12])=> torch.Size ([4, 2166, 12]).

How can this function be transformed into a compatible form of tensorRT instead of "reshape" or "view"?

I don't understand why the project of other developers can work because the train code by pytorch is not compatible with the inference code of tensorRT. Can you fix this problem in the future update or tell me how to solve it?

Thank you!

**Comment by ssmallDream** [comment]
My tensorRT is last version. Why does this project have this obvious disadvantages? Isn't this project running on the Jetson platform using tensorRT acceleration? 

I'm trying to figure out what's wrong, maybe I am not good enough in it.
